scripts/ResourceAutomation/automationAccount.bicep

```diff
@@ -30,6 +33,12 @@ param scheduleStartOfDayDateTime string = '${dateTimeAdd(utcNow(), 'P1D', 'yyyy-
 @description('End of day datetime for schedule')
 param scheduleEndOfDayDateTime string = '${dateTimeAdd(utcNow(), 'P1D', 'yyyy-MM-dd')}T${scheduleEndOfDayTime}-06:00'
 
+@description('Firewall start of day datetime for schedule')
+param firewallScheduleStartOfDayTime string='08:00:00'
```

Comment:
can we change the timing between firewall and cluster/gateway to be more than 10 minutes? Ideally 30 to 60 minutes. Also please add item to board to investigate how chain these two together such that we don't need separate schedules.